### PR TITLE
[res] Add recipe to test FusePreActivationBatchNormPass

### PR DESCRIPTION
**Pull request by jinevening**
This adds a recipe to rest FusePreActivationBatchNormPass

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #4773

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Net_Preactivation_BN_000/test.recipe [hunk 1]
```diff
@@ -0,0 +1,176 @@
+operand {
+  name: "Hole"
+  type: FLOAT32
+  shape {
+    dim: 64
+    dim: 12
+    dim: 12
+    dim: 64
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Preactivation_BN_000/test.recipe:8]: if possible, can you use a small number for testing? current numbers can slow the test and CI

Comment (jinevening) [reply res/TensorFlowLiteRecipes/Net_Preactivation_BN_000/test.recipe]: I've reduced the dimension.

Model size decreased from 2.4 MB to 19.9 kB.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**